packages/mobile/src/components/CreateCommunity/CreateCommunity.component.tsx

```diff
@@ -12,6 +12,7 @@ export const CreateCommunity: FC<CreateCommunityProps> = ({
   createCommunityAction,
   redirectionAction,
   networkCreated,
+  ready = true,
```

Comment:
Why provide that props with default value ?

Reply:
Because we don't want to risk blocking a component without specified reason eg. in case selector failed